our-process/review-timeline.md

```diff
@@ -6,14 +6,14 @@ from start to finish.
 
 ### Step 0. *optional* : Author submits pre-submission inquiry
 A **presubmission inquiry** is useful if you are unsure whether your package
-is in scope. To submit a pre-submission inquiry, open up an issue using the presubmission template in our [pyopensci/software-review repository](https://github.com/pyOpenSci/software-review/issues/new/choose/). During this time an Editor in Chief will review for scope and performs
-basic checks for package infrastructure
+is in scope. To submit a pre-submission inquiry, open up an issue using the presubmission template in our [pyopensci/software-review repository](https://github.com/pyOpenSci/software-review/issues/new/choose/). During this time an Editor-in-Chief will review for scope and perform
+a basic check for package infrastructure.
 
 - Estimated time: ~1-2 weeks
 
 **Below, are the basic checks that your package should have prior to being
-submitted for peer review.** These are the checks that an Editor in Chief or
-and editor will look at when evaluating your package for review.
+submitted for peer review.** These are the checks that an Editor-in-Chief and/or
+editor will look at when evaluating your package for review.
```

Comment:
Good question. To be honest that didn't cross my mind when first reading. The capitalisation of the E-i-C was a gimme as I had in my head a single individual occupying that role. However, no good reason why the other roles shouldn't be capitalised if that is the preference. I can go back and edit if you let me know the best way of proceeding.

Reply:
I would like @lwasser's input on this one. Now that we have an editorial team, the E-i-C might not be a single individual

Reply:
oh gosh ... hmmm it is true EIC may become a team as we grow. @NickleDave seems to be really good with grammar. David, any thoughts on capitalizing roles? we should be consistent. 

ChatGPT says we should capitalize them all (i couldn't resist 😆 )

```
In general, job titles and roles are capitalized when they are used as specific titles before a person's name or when they are used as part of a formal title. However, when they are used generically or descriptively, they are not capitalized.
```

great catch on this @Batalex

Reply:
Sorry all. Should I go ahead and capitalise all the roles for consistency as suggested? If so, I will go back in and make the changes in hopes of wrapping this one up quickly.

Reply:
sure - let's go ahead and capitalise all roles! Then we can merge this! thank you @g-patlewicz for your persistence with this pr. i've been offline for the past 2 weeks and am just catching up!!

Reply:
At the risk of being pedantic ... I think you were right above @lwasser that we should not capitalize when these titles are used generally, instead of in front of a specific person's names

> However, when they are used generically or descriptively, they are not capitalized.

So I think we should just **not** capitalize "editor in chief".

And just to be extra pedantic, I think we don't want to hyphenate.
See discussion here https://en.wikipedia.org/wiki/Talk%3AEditor-in-chief
and example here: https://www.dictionary.com/browse/editor-in-chief